public/index.php

```diff
@@ -1,4 +1,5 @@
 <?php
+define('API_URL', 'http://hercules.code4hr.org');
```

Comment:
Since the API_URL environment variable is set in the [dockerfile] (https://github.com/Code4HR/explore/blob/master/Dockerfile#L20) wouldn't we use getenv('API_URL') instead of hardcoding the URL?